modules/config/URLProcessor.js

```diff
@@ -1,69 +1,94 @@
-/* global config, getConfigParamsFromUrl, interfaceConfig, loggingConfig */
+/* global config, interfaceConfig, loggingConfig */
 
 const logger = require("jitsi-meet-logger").getLogger(__filename);
 
-var configUtils = require('./Util');
-var params = {};
+import { getConfigParamsFromUrl } from '../../react/features/base/config';
 
-if (typeof getConfigParamsFromUrl === 'function') {
-    params = getConfigParamsFromUrl();
-}
+import configUtils from './Util';
 
-var URLProcessor = {
-    setConfigParametersFromUrl: function () {
-        // Convert 'params' to JSON object
-        // We have:
-        // {
-        //   "config.disableAudioLevels": false,
-        //   "config.channelLastN": -1,
-        //   "interfaceConfig.APP_NAME": "Jitsi Meet"
-        // }
-        // We want to have:
-        // {
-        //   "config": {
-        //     "disableAudioLevels": false,
-        //     "channelLastN": -1
-        //   },
-        //   interfaceConfig: {
-        //     APP_NAME: "Jitsi Meet"
-        //   }
-        // }
-        var configJSON = {
+// Parsing config params from URL hash.
+const URL_PARAMS = getConfigParamsFromUrl(window.location);
+
+/**
+ *  URL params with this prefix should be merged to config.
+ */
+const CONFIG_PREFIX = 'config.';
+
+/**
+ *  URL params with this prefix should be merged to interface config.
+ */
+const INTERFACE_CONFIG_PREFIX = 'interfaceConfig.';
+
+/**
+ *  URL params with this prefix should be merged to logging config.
+ */
+const LOGGING_CONFIG_PREFIX = 'loggingConfig.';
+
+/**
+ * Config keys to be ignored.
+ *
+ * @type Set
+ */
+const KEYS_TO_IGNORE = new Set([
+    'analyticsScriptUrls',
+    'callStatsCustomScriptUrl'
+]);
+
+/**
+ * Convert 'URL_PARAMS' to JSON object
+ * We have:
+ * {
+ *      "config.disableAudioLevels": false,
+ *      "config.channelLastN": -1,
+ *      "interfaceConfig.APP_NAME": "Jitsi Meet"
+ * }
+ * We want to have:
+ * {
+ *      "config": {
+ *          "disableAudioLevels": false,
+ *          "channelLastN": -1
+ *      },
+ *      interfaceConfig: {
+ *          "APP_NAME": "Jitsi Meet"
+ *      }
+ * }
+ */
+export default {
+    setConfigParametersFromUrl() {
```

Comment:
`setConfigParametersFromUrl` is basically the one and only static function in what used to be called `URLProcessor` and is not called `default`. There's no benefit to the existence of `URLProcessor`/`default` so a simple `export function setConfigParametersFromUrl` will suffice in this ES6 module. Moreover, that'd be in line with the approach that we've adopted in react/ where this contents of this file will have to eventually land.